contrib/cirrus/win-podman-machine-test.ps1

```diff
@@ -33,5 +33,12 @@ Run-Command ".\bin\windows\podman.exe --version"
 New-Item -ItemType "directory" -Path "$env:AppData\containers"
 Copy-Item -Path pkg\machine\ocipull\policy.json -Destination "$env:AppData\containers"
 
+# Install Perl
+Write-Host "`nInstalling Perl as it is required to use logformatter"
+Run-Command 'choco install --no-progress --confirm --acceptlicense --nocolor StrawberryPerl'
```

Comment:
I guess that should be moved to our image builds: https://github.com/containers/automation_images/blob/a3e4099c72052ff21a6e538043cf05b5d873a110/win_images/win_packaging.ps1#L12

Reply:
Done but untested: https://github.com/containers/automation_images/pull/337#issuecomment-2005057595

Reply:
Currently we are checking if Perl exists before invoking logformatter and install it if necessary:

https://github.com/containers/podman/blob/fb046fc24a16af87357a7f1335661c88691e2f40/contrib/cirrus/win-lib.ps1#L43-L50

I will open another PR to remove this code as soon as Perl is available in the new windows image.

Reply:
#22248